src/main/java/nl/novi/techiteasy1121/controllers/TelevisionController.java

```diff
@@ -43,52 +46,63 @@ public ResponseEntity<List<TelevisionDto>> getAllTelevisions(@RequestParam(value
 
     }
 
-    // De return waarde is ook hier een TelevisionDto in plaats van een Television
     @GetMapping("/televisions/{id}")
     public ResponseEntity<TelevisionDto> getTelevision(@PathVariable("id")Long id) {
 
-        // We spreken hier ook weer een service methode aan in plaats van direct de repository aan te spreken
         TelevisionDto television = televisionService.getTelevisionById(id);
 
             return ResponseEntity.ok().body(television);
 
     }
 
-    // Ook hier returnen we weer een TelevisionDto, maar ook de parameter is een dto geworden.
-    // Het is niet verplicht om een "outputdto" en een "inputdto" te hebben, zeker omdat ze in dit geval hetzelfde zijn,
-    // maar we willen jullie laten zien dat het mogelijk is. In sommige gevallen kan het zelfs nodig zijn.
     @PostMapping("/televisions")
-    public ResponseEntity<TelevisionDto> addTelevision(@Valid @RequestBody TelevisionInputDto televisionInputDto) {
+    public ResponseEntity<Object> addTelevision(@Valid @RequestBody TelevisionInputDto televisionInputDto) {
 
-        // Hier gebruiken we weer een service methode in plaats van direct de repository aan te spreken.
         TelevisionDto dto = televisionService.addTelevision(televisionInputDto);
 
         return ResponseEntity.created(null).body(dto);
 
     }
 
-    // Hier veranderd niks aan de methode. We hebben niet meer de naam van de pathvariabele expliciet genoemd, omdat de
-    // parameter-naam overeen komt met de naam van de pathvariabele.
     @DeleteMapping("/televisions/{id}")
     public ResponseEntity<Object> deleteTelevision(@PathVariable Long id) {
 
-        // Hier gebruiken we weer een service methode in plaats van direct de repository aan te spreken.
         televisionService.deleteTelevision(id);
 
         return ResponseEntity.noContent().build();
 
```

Comment:
Zorg ervoor dat je de juiste REST principes volgt, zoals het gebruik van de juiste HTTP methoden en status codes. Dit is cruciaal voor een goede API-ontwikkeling.